db/models.js

```diff
@@ -1,46 +1,55 @@
-const sequelize = require('./index.js');
+const ORM = require('./index.js');
 
-// will implement use of the authorId variable
-// let authorId = '';
-
-exports.getAuthorInfo = (bookId, callback) => {
-  const authorQuery = `SELECT name, followers, biography FROM authors WHERE id IN (SELECT author_id FROM books WHERE id = ${bookId})`;
-  sequelize.sequelize.query(authorQuery)
-    .then(([results]) => {
-      // set authorId (from line 3) to be passed into getFiveBooks
-      callback(null, results[0]);
-    })
-    .catch((err) => {
-      console.log('err', err);
-    });
-};
-
-exports.getFiveBooks = (authorId, callback) => {
+const getFiveBooks = (authorId, callback) => {
   console.log('authorId', authorId);
-  const fiveBooksQuery = `SELECT title FROM books WHERE author_id = ${authorId} ORDER BY average_ratings LIMIT 5`;
-  sequelize.sequelize.query(fiveBooksQuery)
+  const fiveBooksQuery = `SELECT title FROM books WHERE author_id = ${authorId} ORDER BY average_rating LIMIT 5`;
```

Comment:
you need to exclude the book you're currently viewing on the page from this query. `AND id != {$id}`

Reply:
also nevermind, looks like you're withdrawing this endpoint

Reply:
Nice catch! I did NOT think of this.